tests/queries/0_stateless/03008_azure_plain_rewritable.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-shared-merge-tree
+# Tag no-fasttest: requires Azure
+# Tag no-shared-merge-tree: does not support replication
+
+set -e
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+container="cont-$(echo "${CLICKHOUSE_TEST_UNIQUE_NAME}" | tr _ -)"
+
+${CLICKHOUSE_CLIENT} --query "drop table if exists test_azure_mt"
+
+${CLICKHOUSE_CLIENT} -nm --query "
+create table test_azure_mt (a Int32, b Int64, c Int64) engine = MergeTree() partition by intDiv(a, 1000) order by tuple(a, b)
+settings disk = disk(
+    type = object_storage,
+    metadata_type = plain_rewritable,
+    object_storage_type = azure_blob_storage,
+    name = '${container}',
+    path='/var/lib/clickhouse/disks/${container}/tables',
+    container_name = '${container}',
+    endpoint = 'http://localhost:10000/devstoreaccount1/${container}/plain-tables',
```

Comment:
in production cloud we define the `endpoint` with a `/` at the end, this is i think causing an exception 

https://clickhouse-inc.slack.com/archives/C02F2LML5UG/p1733233064721519?thread_ts=1719249147.971119&cid=C02F2LML5UG